README.md

```diff
@@ -95,12 +96,12 @@ pip install -e ".[dev]"
             Rest of the doc
             rest of the doc
         </doc>
-    </field>
-    <field name="velocity">
-      <doc>
-           A single block of doc string.
-      </doc>
-    </field>
+      </field>
```

Comment:
Having that indentation does no harm

Reply:
It broke the display of the code block. Thus it needed to be changed